_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Social Media Use in 2021"
+description: |
+  % of adults using each social media.
+categories: "2022" 
+author: Álvaro Sanz 
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(fig.showtext=TRUE)
+
+
+library(tidyverse)
+library(ggplot2)
+library(lemon)
+library(showtext)
+
+```
+
+
+## Introduction
+
+### Getting the data
+
+```{r}
+
+data <- read_delim("Data/database.csv")
+
+data <- data %>% rename(
+  "Social" = `Social media`
+)
+
+filtered <- data %>% 
+  group_by(Social) %>% 
+  filter(Year==min(Year)|Year==max(Year))
+
+```
+
+### Font preparation
+```{r}
+
+font_add_google(name = "League Gothic", family = "League Gothic")
+font_add(family = "Georgia", regular = "GEORGIAI.ttf")
+showtext_auto()
+showtext.opts(dpi = 150)
```

Comment:
I don't think you need these once you set `fig.showtext=TRUE` above. The issue could be that fonts from files and from Google may not play well maybe? Try using another font from Google Fonts substituting Georgia to see if this makes any difference.